getting_started/configure_openshift.adoc

```diff
@@ -0,0 +1,237 @@
+[[getting-started-configure-openshift]]
+= Configure {product-title}
+:data-uri:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[configure-overview]]
+== Overview
+
+This guide introduces you to the basic concepts of {product-title}, and helps
+you configure a basic application. This guide is not suitable for deploying or
+installing a production environment of OpenShift.
+
+[[change-log-in-identity-provider]]
+== Change Log In Identity Provider
+
+The default behavior of a freshly installed {product-title} instance is to deny
+any user from logging in. To change the authentication method to HTPasswd:
+
+. Open the *_/etc/origin/master/master-config.yaml_* file in edit mode.
+. Find the `identityProviders` section.
+. Change
+`DenyAllPasswordIdentityProvider` to `HTPasswdPasswordIdentityProvider`
+provider.
+. Change the value of the name label to `htpasswd_auth` and add a
+new line `file: /etc/origin/openshift-passwd` in the provider section.
++
+An example `identityProviders` section with `HTPasswdPasswordIdentityProvider`
+would look like the following:
++
+----
+identityProviders: - challenge: true login: true
+name: htpasswd_auth provider: apiVersion: v1 file: /etc/origin/openshift-passwd
+kind: HTPasswdPasswordIdentityProvider
+----
+
+. Save the file.
+
+[[create-user-accounts]]
+== Create User Accounts
+
+Now that you are using the `HTPasswdPasswordIdentityProvider` provider, you
+need to generate these user accounts.
+
+. You can use the *httpd-tools* package to obtain the *htpasswd* binary that can
+generate these accounts.
++
+----
+yum -y install httpd-tools
+----
+
+. Create a user account.
++
+----
+touch /etc/origin/openshift-passwd htpasswd -b
+/etc/origin/openshift-passwd admin redhat
+----
++
+Note that you have created a
+user with the username of `admin` and password of `redhat`.
```

Comment:
It might be more accurate to link the user and their password more. Maybe something like "...you have created a user named `admin` with a password of `redhat`."